### ISSUE TITLE
PGGAN generator has some bug?

### ISSUE THREAD

**Issue by kuoluo1995** [kozistr/Awesome-GANs#13]
I find PGGAN can not train.
When pg ==1, generator make the fake image which shape is [-1,16,16, 3].
And i find reason: because generator has upsample twice on line of block the pggan_model.py .
repair
```
x = block(x, nf(i + 1), name="1")
# x = block(x, nf(i + 1), name="2")
x = tf.layers.conv2d(x, filters=get_out_channels(i + 1), kernel_size=3, strides=1, padding='SAME',
                        kernel_initializer=w_init, kernel_regularizer=w_reg, bias_initializer=b_init,
                        name='gen_n_%s_conv2d-%d' % (name, x.get_shape()[1]))
x = tf.nn.leaky_relu(x)
x = pix_norm(x)
```

**Comment by kuoluo1995** [comment]
And orthogonal matrix is very Important.
Without it, you will be Exploding Gradients when process grow size in 6.

**Comment by kuoluo1995** [comment]
And initializer is very important.
Your method will be exploding gradients when pg = 6.

**Comment by kozistr** [comment]
thx for reporting the issue!

some of my implementations are not maintained and there're several issues like PGGAN impl of mine (not perfectly impled) :(

someday, i'll fix the issues as possible as i can.
thank you in advance!
